src/test/java/uk/gov/dhsc/htbhf/CucumberConfiguration.java

```diff
@@ -14,15 +16,19 @@
 public class CucumberConfiguration {
 
     @Bean(destroyMethod = "close")
-    public WebDriver webDriver(@Value("${test.browser}") String browser) {
+    public WebDriver webDriver(@Value("${test.browser}") String browser, @Value("${test.headless}") boolean headless) {
         WebDriver webdriver = null;
         switch (browser) {
             case "firefox":
+                FirefoxOptions firefoxOptions = new FirefoxOptions();
+                firefoxOptions.setHeadless(headless);
                 webdriver = new FirefoxDriver();
```

Comment:
Do we need to pass in the firefoxOptions here?

Reply:
Yep, fixed.